_assets/localhost_config.yml

```diff
@@ -0,0 +1,2 @@
+url: "" # blank in development
```

Comment:
this fixed the api issue when viewing locally?

Reply:
it needs a gulp task that looks here for the config that I haven't set up yet. i'll pull this for now and add both in if it's needed.